src/rest/Facet/FacetSearchRequest.ts

```diff
@@ -2,14 +2,48 @@ import { IQuery } from '../Query';
 
 /**
  * A Search API facet search request.
- *
- * TODO: documentation
  */
 export interface IFacetSearchRequest {
+  /**
+   * The name of the field against which to execute the facet search request.
+   */
   field: string;
+  /**
+   * A list of index field values to filter out from the facet search results.
+   *
+   * **Example:**
+   * > `["blue", "green"]`
```

Comment:
In general, you can remove the ">" and put the sample on the same line as **Example:**. I.e.,

```
**Example:** `["blue", "green"]`
```